contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/AbstractTokenContract.kt

```diff
@@ -122,10 +122,6 @@ abstract class AbstractTokenContract<AT : AbstractToken> : Contract {
 			}
 			dispatchOnCommand(commands, group.inputs, group.outputs, tx.attachments, tx.references)
 		}
-
-
-		val allMatchedCommands = groupsAndCommands.map { it.first.first() }.toSet()
```

Comment:
Why have these been removed?

Reply:
It is the just assignment and this val is not used anywhere. So there is compiler warning because of the used variable.